test/cpp/qps/usage_timer.cc

```diff
@@ -66,6 +67,9 @@ static void get_cpu_usage(unsigned long long* total_cpu_time,
       *idle_cpu_time = std::stol(cpu_time_str);
     }
   }
+#else
+  gpr_log(GPR_INFO, "get_cpu_usage(): Non-linux platform is not supported.").
+#endif
```

Comment:
Rather than setting those variables to 0 in the caller and then not changing them here, I would suggest letting those values be uninitialized in the caller and then setting them to 0 here.